features/steps/steps.go

```diff
@@ -55,6 +62,25 @@ func (c *Component) theDocumentInTheDatabaseShouldBe(id string, doc *godog.DocSt
 	return nil
 }
 
+func (c *Component) iHaveTheseFilters(docs *godog.DocString) error {
+	ctx := context.Background()
+	var filters []model.Filter
+	m := c.store
+
+	err := json.Unmarshal([]byte(docs.Content), &filters)
+	if err != nil {
+		return err
```

Comment:
Could you wrap these errors with some context? `errors.Wrap(err, "failed to blah")` etc. Thanks

Reply:
this has been addressed.